.github/workflows/tests.yml

```diff
@@ -5,39 +5,38 @@ name: Tests
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   test:
-
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04]
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Install Ubuntu dependencies
-      run: >
-        sudo apt-get update -q && sudo apt-get install
-        --no-install-recommends -y xvfb python3-dev python3-gi
-        python3-gi-cairo gir1.2-gtk-3.0 libgirepository1.0-dev
-        gir1.2-poppler-0.18 libcairo2-dev enchant python3-enchant intltool
-    - name: Install dependencies
-      run: |
-        sudo python3 -m pip install --upgrade pip
-        sudo pip3 install flake8 nose mypy
-        sudo pip3 install --upgrade keyrings.alt
-        if [ -f requirements.txt ]; then pip3 install -r requirements.txt; fi
-    - name: Prepare plugins
-      run: python3 setup.py build_i18n
-    - name: Test with nose
-      run: |
-        xvfb-run -a nosetests -v gourmet/tests/test_*
-    - name: Test with mypy
-      run: |
-        mypy gourmet/
-      if: always()
+      - uses: actions/checkout@v2
+      - name: Install Ubuntu dependencies
+        run: >
+          sudo apt-get update -q && sudo apt-get install
+          --no-install-recommends -y xvfb python3-dev python3-gi
+          python3-gi-cairo gir1.2-gtk-3.0 libgirepository1.0-dev
+          gir1.2-poppler-0.18 libcairo2-dev enchant python3-enchant intltool
+      - name: Install dependencies
+        run: |
+          sudo python3 -m pip install --upgrade pip
+          if [ -f development.txt ]; then pip3 install -r development.txt; fi
```

Comment:
This should be `...; then sudo pip3 install -r development.txt; fi` because otherwise, the packages are installed in the user directory `/home/runner/.local/bin` which is not set in the PATH.
That's why the tests failed just now:

```
  xvfb-run -a nosetests -v gourmet/tests/test_*
  shell: /usr/bin/bash -e {0}
/usr/bin/xvfb-run: 184: nosetests: not found
##[error]Process completed with exit code 127.
```

The libraries can be installed as user, because Python knows where to look for them, although it would not hurt to install them as `sudo`, as then it's closer to a real user installation :)